utils/logalloc.cc

```diff
@@ -204,6 +204,34 @@ migrate_fn_type::unregister_migrator(uint32_t index) {
     static_migrators().remove(index);
 }
 
+
+namespace {
+
+// for printing extra message in reclaim_timer::report when stall is detected
```

Comment:
Please add a comment explaining *why* this struct/formatter is needed.

The purpose is to ensure no dynamic allocations in a function involved in handling OOMs. This code doesn't make this obvious to the reader.